test/functional/test_site_special_tags/_markbind/plugins/testSpecialTag.js

```diff
@@ -17,11 +23,20 @@ function preRender(content) {
     $(testElement).text(wrappedText);
   });
 
+  const escapedNunjucks = $('mustache');
+  escapedNunjucks.each((index, element) => {
+    const unwrappedText = $(element).text();
+    const unescapedText = unwrappedText.replace(ESCAPE_REGEX, removeEscapeTags);
+    const transformedText = unescapedText.replace(/{/g, '!success').replace(/}/g, 'success!');
```

Comment:
Shouldn't the text transformation ( `!success ... success!` ) be done in `removeEscapeTags`?
Text without `{% raw %}` would still be transformed otherwise

Reply:
The previous statement is to remove `{% raw %}` for this test. This can be simplified if we have some attr like `escape` for plugins which automatically removes these raw statements. This line is to replace the mustaches `{{` to success statements for this specific test. In normal circumstance, text with/without `{% raw %}` outside of special tags would not have `{{` available during post-render since `{% raw %}` would be removed by markdown-it-attr.

Reply:
Ah I see, how about wrapping it with `unescapedText = unwrappedText.replace(ESCAPE_REGEX, 'raw$1endraw')` to ensure {% raw %} was present at the time of `postRender`? ( since `replace` would return the original string if there's no match as well )

Reply:
That makes things clearer :+1: